src/deepsparse/engine.py

```diff
@@ -23,13 +23,19 @@
 import numpy
 
 from deepsparse.benchmark import BenchmarkResults
+from tqdm.auto import tqdm
```

Comment:
tqdm needs to be added to known third party in the setup.cfg and since we're explicitly importing it here I think we should add it as an explicit dependency in the setup.py for a specific version range